src/typeahead/typeahead.ts

```diff
@@ -57,7 +57,8 @@ export class NgbTypeahead implements OnInit,
   private _popupService: PopupService<NgbTypeaheadWindow>;
   private _positioning = new Positioning();
   private _valueChanges = new Subject<string>();
-  private _windowRef: ComponentRef<NgbTypeaheadWindow>;
+
+  windowRef: ComponentRef<NgbTypeaheadWindow>;
```

Comment:
Hah! I just blindly followed your carousel example, to be honest. You made keyPrev() and keyNext() public, and they're also only used in host :-)